src/librustdoc/clean/mod.rs

```diff
@@ -2283,18 +2283,19 @@ impl Clean<Item> for (&hir::MacroDef<'_>, Option<Symbol>) {
         } else {
             let vis = item.vis.clean(cx);
             let def_id = item.def_id.to_def_id();
+            let relative_to = vec![];
```

Comment:
This looks incorrect, decl macros are not necessarily at the crate root: https://github.com/rust-lang/rust/pull/77862. You should be able to get the parent with `tcx.hir().parent_module(tcx.hir().local_def_id_to_hir_id(item.def_id))`.

Reply:
You're right.

Fixed in d85b42ab3339b12b065ec9df75270ab232f88991